### PR TITLE
js functionality added in both pages

### PR DESCRIPTION
**Pull request by Svartstein**
# Imtortant!
 This project uses Bootstrap 5, please install to display the pages correctly.

### JS Functions added for the home and about the pages:
Creates a list of speakers dinamically and manipulates that list in the home page.
Manipulates the mobile menu from the hamburger menu.

### Previous pull request:
- Were created the home and about pages
- Was added their respective styles with Bootstrap 5 and my own css.

### Video Link:
https://drive.google.com/file/d/1RA2RLYQkh4YM6ZDvtryAswTmXHHzqT2O/view?usp=sharing

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @Swarzstein 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project, that has Linter errors (Eslint), kindly fix them 👌 
Linter result should be green, this means that the last commit have a green check mark (✅)

- [x] Please update the summary description for this PR, it should be more descriptive 👌

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -26,33 +26,42 @@
       <a class="py-2 px-4" onclick="displayMenu()" href="#"><h2>CTF Campaigns</h2></a>
     </div>
   </div>
-
+  
+  <button id="menu-button" class="hamburger mobile-only absolute" type="button">
+      <div></div><div></div><div></div>
```
Comment (emmiiorji) [inline index.html:31]: Kindly remove unused tags to clean up the code. You'll agree that the div tags highlighted above seem to be dormant.

File: styles.css [hunk 2]
```diff
@@ -263,8 +307,13 @@ span {
...
+  .dark-desktop {
+    background: #2e2e2e !important;
+    color: #aeaeae;
+  }
+
   .mobile-only {
-    display: none;
+    display: none !important;
   }
 }
```
Comment (emmiiorji) [inline styles.css:319]: Kindly include adequate styling for the desktop version (and mobile version too) of the project in order to correspond to the design guideline.

Your design:
image
image
image
image
image


Design guideline:
image
image
image
image

File: README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (emmiiorji) [inline README.md]: Please, include in your README an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @Swarzstein, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Almost there

## Good Highlights✔

- Linter checks are passing
- Live URL is working properly
- Good README documentation


## Required Changes ♻️

- As specified in the requirements, kindly record a video of maximum length 5mins where you present the features of the project and highlight about a piece of the code the student finds interesting or is proud of making.
- Kindly follow the design guideline (both mobile and desktop versions) and reproduce it as much as possible while personalizing the content 
- _Check the comments under the review for more required changes._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Svartstein) [reply index.html:31]: this tags are used to create the hamburger button

File: styles.css (hunk 2 above)
Comment (Svartstein) [reply styles.css:319]: It was clearly said that Bootstrap had to be installed to display the proyect correctly, I even said this in the submition.

File: README.md (hunk 3 above)
Comment (Bondok6) [reply README.md]: - [x] Your README is well documented 💯 but I would suggest giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :wink: :pushpin:

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @Swarzstein 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## To Highlight 
- Your project is professional ✅
- Great job using semantic tags ✅
- No linter errors ✅
- Your repo is professional ✅

## Suggested changes
- [x] As the previous reviewer mentioned and according to the requirements, you should record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

image
- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 4]
```diff
@@ -1 +1,48 @@
...
+
+- [Demo video](https://drive.google.com/file/d/1RA2RLYQkh4YM6ZDvtryAswTmXHHzqT2O/view?usp=sharing)
+
+## Prerequisites
+
+- Needs Bootstrap 5 installed in the folder in order to display it correctly.
+
+### Deployment
+
+https://swarzstein.github.io/Conference-Page/
```
Comment (Bondok6) [inline README.md:22]: - [OPTIONAL] I appreciate that you implemented the changes 💯 
I noticed that the deploy link is broken, kindly fix it.

> But for the hard work you've done, your project deserves to be approved 🎉 I am sure that you will fix this minor issue

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Swarzstein 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Good video presentation ✅💯
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 2 above)
Comment (emmiiorji) [reply styles.css:319]: Hi @Swarzstein,

That was a costly oversight. Sorry about that.

You put in a great deal of effort; well done.

**Pull request merged**